CityOfDetroit/bloom
# fix: update styling of coming soon tag

## Issue

- Closes #1074 (QA)

## Description

Changes the styling and text of the `Coming Soon` tag following QA. I'd recommend a PR back to dev for the ui-component changes.

## How Can This Be Tested/Reviewed?

Add a listing that has the marketing type of coming soon with a season and a date, ensure the tag appears
Add a listing that is verified, ensure the tag appears
Add a listing that is both coming soon and verified, ensure only coming soon appears

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
